packages/fx-core/src/component/generator/officeXMLAddin/projectConfig.ts

```diff
@@ -1,208 +1,189 @@
+    },
   },
 };
 
-const OfficeXMLAddinProjectConfig: IOfficeXMLAddinProjectConfig = {
+export const OfficeAddinProjectConfig: IOfficeAddinProjectConfig = {
+  json: {
+    "json-taskpane": {
+      title: "core.newTaskpaneAddin.label",
+      detail: "core.newTaskpaneAddin.detail",
+      localTemplate: "",
+      framework: {
+        default_old: {
+          typescript: "https://aka.ms/teams-toolkit/office-addin-taskpane",
+        },
+        default: {
+          typescript: "https://aka.ms/teams-toolkit/office-addin-taskpane/ts-default",
+          javascript: "https://aka.ms/teams-toolkit/office-addin-taskpane/js-default",
+        },
+        react: {
+          typescript: "https://aka.ms/teams-toolkit/office-addin-taskpane/ts-react",
+          javascript: "https://aka.ms/teams-toolkit/office-addin-taskpane/js-react",
+        },
+      },
+      manifestPath: "manifest.json",
+    },
+  },
   word: {
-    taskpane: {
+    "word-taskpane": {
       localTemplate: "word-taskpane",
       ...CommonProjectConfig.taskpane,
     },
-    sso: {
+    "word-sso": {
       title: "core.createProjectQuestion.officeXMLAddin.word.sso.title",
       detail: "core.createProjectQuestion.officeXMLAddin.word.sso.detail",
       localTemplate: "word-sso",
       ...CommonProjectConfig.sso,
     },
-    react: {
+    "word-react": {
       title: "core.createProjectQuestion.officeXMLAddin.word.react.title",
       detail: "core.createProjectQuestion.officeXMLAddin.word.react.detail",
       localTemplate: "word-react",
       ...CommonProjectConfig.react,
     },
-    manifest: {
+    "word-manifest": {
       localTemplate: "word-manifest-only",
       ...CommonProjectConfig.manifest,
     },
   },
   excel: {
-    taskpane: {
+    "excel-taskpane": {
       localTemplate: "excel-taskpane",
       ...CommonProjectConfig.taskpane,
     },
-    sso: {
+    "excel-sso": {
       title: "core.createProjectQuestion.officeXMLAddin.excel.sso.title",
       detail: "core.createProjectQuestion.officeXMLAddin.excel.sso.detail",
       localTemplate: "excel-sso",
       ...CommonProjectConfig.sso,
     },
-    react: {
+    "excel-react": {
       title: "core.createProjectQuestion.officeXMLAddin.excel.react.title",
       detail: "core.createProjectQuestion.officeXMLAddin.excel.react.detail",
       localTemplate: "excel-react",
       ...CommonProjectConfig.react,
     },
-    cfShared: {
+    "excel-cfshared": {
       title: "core.createProjectQuestion.officeXMLAddin.excel.cf.shared.title",
       detail: "core.createProjectQuestion.officeXMLAddin.excel.cf.shared.detail",
       localTemplate: "excel-cf",
-      lang: {
-        ts: "https://aka.ms/ccdevx-fx-cf-shared-ts",
-        js: "https://aka.ms/ccdevx-fx-cf-shared-js",
+      framework: {
+        default: {
+          typescript: "https://aka.ms/ccdevx-fx-cf-shared-ts",
+          javascript: "https://aka.ms/ccdevx-fx-cf-shared-js",
+        },
       },
     },
-    cfJS: {
+    "excel-cfjs": {
```

Comment:
Actually, cfshared is stand for "Custom Functions with Shared Runtime"\
cfjs is stand for "Custom Functions with JavaScript Runtime"

When used in CLI options, maybe we can consider rename this capability type?

Reply:
It's OK to use capability id here, CLI has a command to explain the capability description.